bundles/pubsub/pubsub_spi/include/pubsub_protocol.h

```diff
@@ -32,27 +33,29 @@ typedef struct pubsub_protocol_header pubsub_protocol_header_t;
  * The protocol header structure, contains the information about the message payload and metadata
  */
 struct pubsub_protocol_header {
-  /** message payload identification attributes */
-    unsigned int msgId;
-    unsigned short msgMajorVersion;
-    unsigned short msgMinorVersion;
+    /** message payload identification attributes */
+    uint32_t msgId;
+    uint16_t msgMajorVersion;
+    uint16_t msgMinorVersion;
 
     /** Payload and metadata sizes attributes */
-    unsigned int payloadSize;
-    unsigned int metadataSize;
+    uint32_t payloadSize;
+    uint32_t metadataSize;
 
     /** Optional message segmentation attributes, these attributes are only used/written by the protocol admin.
      *  When message segmentation is supported by the protocol admin */
-    unsigned int seqNr;
-    unsigned int payloadPartSize;
-    unsigned int payloadOffset;
+    uint32_t seqNr;
+    uint32_t payloadPartSize;
+    uint32_t payloadOffset;
+
+    uint32_t padding; //to arrange alignment on 64 bit
```

Comment:
What is the added value of this?

Reply:
So that when addressing the payload in memory (if they are read at the same time) the alignment is 64 bit. 

If I am correct this is not needed, but can help in performance. 
@rbulter: Do you know if this is really needed of an overdone performance feature/complexity?

Reply:
I would expect the compiler to do something here, if needed at all... But don't know for sure.

Reply:
Probably better to use the aligned attribute: https://stackoverflow.com/a/11130216

Reply:
Padding was added because we still same problems in the pubsub admins. This is not due to alignment so that is not needed. wil revert his.